extensions/wikia/Chat2/tests/ChatWidgetTest.php

```diff
@@ -112,7 +111,6 @@ public function testGeTemplateVarsDataProvider() {
 				'expected' => [
 					'blankImgUrl' => 'www.url.com',
 					'buttonText' => 'message',
-					'buttonIcon' => '<svg></svg>',
 					'guidelinesText' => 'message',
 					'fromParserTag' => true,
 					'sectionClassName' => 'ChatWidget',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sectionClassName" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)